package.json

```diff
@@ -36,7 +36,7 @@
     "licenseCheck:run": "license-tool/run.sh"
   },
   "dependencies": {
-    "@eclipse-che/workspace-client": "^0.0.1-1612278649",
```

Comment:
note that I remove ^ since we need to avoid auto upgrade of dependencies, let me know if I'm wrong